doc/troubleshooting.md

```diff
@@ -1024,6 +1053,28 @@ In case the alarm is still raised after calibration was performed, TF broadcasti
 Open a new issue on the [Issue tracker](https://github.com/yaskawa-global/motoros2/issues), describe the problem and attach `PANELBOX.LOG`, `RBCALIB.DAT` and the debug log to the issue.
 Include a verbatim copy of the alarm text as seen on the teach pendant (alarm number and `[subcode]`).
 
+### Alarm: 8013[17]
+
+*Example:*
+
+```text
+ALARM 8013
+ Bad UserLan debug port in cfg
+[17]
+```
+
+*Solution:*
+The `userlan_debug_broadcast_port` key in the `motoros2_config.yaml` configuration file is set to an invalid value.
+Debug broadcasting will be disabled for this session.
```

Comment:
> Debug broadcasting will be disabled for this session

I don't like this behavior. I definitely think broadcasting should be enabled if there's an error in the configuration.

Reply:
I can change that. So `userlan_monitoring_enabled` should be set to disabled if there is an error in the config, but broadcasting should still be enabled? I only ask because they're in the same case for the config switch case.

Reply:
I don't have an opinion on the behavior of `userlan_monitoring_enabled` in this case

Reply:
UserLan monitoring is extra functionality: it makes detecting disconnections faster, but isn't absolutely necessary (other places will also detect disconnects, they'll just take longer).

There's no harm in disabling it if the configuration is incorrect.

But debug broadcasts should indeed not be disabled in cases of incorrect configuration. Especially since we often use debug logs to diagnose misconfiguration.

Reply:
I changed the behavior and this documentation so that it broadcasts to all ports if the specific port isn't active.